packages/panels/resources/views/components/form/actions.blade.php

```diff
@@ -7,20 +7,25 @@
 @if (count($actions))
     <div
         @if ($this->areFormActionsSticky())
-            x-data="{
+        x-data="{
                 isSticky: false,
-
+                isPinned: false,
                 evaluatePageScrollPosition: function () {
-                    this.isSticky =
-                        document.body.scrollHeight >=
-                        window.scrollY + window.innerHeight * 2
+                    this.isSticky = (this.isPinned == false && window.scrollY >= 0);
+                    this.isPinned = $el.getBoundingClientRect().bottom <= window.innerHeight;
                 },
             }"
             x-init="evaluatePageScrollPosition"
+            x-intersect.threshold.100="isPinned = true"
             x-on:scroll.window="evaluatePageScrollPosition"
+            x-bind:style="{
+                bottom: '-1px',
+            }"
```

Comment:
Could you move this after `x-bind:class` and run Prettier to fix formatting issues, @drhoussem?